src/devices/Mcp23xxx/Mcp23xxx.cs

```diff
@@ -205,54 +206,41 @@ public virtual void Dispose()
         {
             _masterGpioController?.Dispose();
             _masterGpioController = null;
-            _device?.Dispose();
+            _bus?.Dispose();
         }
 
         /// <summary>
-        /// Disables the device by setting the reset pin LOW.
+        /// Disables the device by setting the reset pin low.
```

Comment:
Question: Is the convention better to have uppercase or lowercase for things like OFF/ON, LOW/HIGH, FALSE/TRUE, etc.?  Just curious as I want to make sure I'm following the same in other bindings.

Reply:
I'm just leaning towards how it would be written in the docs. We wouldn't write it in caps there.